botfront/cypress/package.json

```diff
@@ -10,16 +10,16 @@
   "author": "Botfront <hi@botfront> ",
   "license": "AGPL",
   "private": true,
-  "devDependencies": {
+  "dependencies": {
+    "@octokit/rest": "^18.0.5",
     "axios": "^0.19.0",
     "chai": "^4.2.0",
-    "cypress": "3.6.0",
-    "cypress-plugin-retries": "1.2.2",
-    "js-yaml": "^3.13.1"
-  },
-  "dependencies": {
     "glob": "^7.1.6",
+    "js-yaml": "^3.13.1",
     "jszip": "^3.5.0",
-    "path": "^0.12.7"
+    "keypair": "^1.0.1",
```

Comment:
This lib has a GPL license 😢  iirc we would have to distribute botfront under GPL to be compliant ? so we can't use this lib @znat ?